### ISSUE TITLE
Requires babel-polyfill since 0.5.1

### ISSUE THREAD

**Issue by wannesdemaeght** [HyperCubeProject/binance-api-node#32]
Updated from 0.5.0 to 0.5.2, trader won't start.
Error:

```
C:\Trader\Logic\node_modules\binance-api-node\dist\http.js:105
    var _ref4 = _asyncToGenerator( /*#__PURE__*/regeneratorRuntime.mark(function _callee(path) {
                                                ^

ReferenceError: regeneratorRuntime is not defined
    at C:\Trader\Logic\node_modules\binance-api-node\dist\http.js:105:49
    at privateCall (C:\Trader\Logic\node_modules\binance-api-node\dist\http.js:168:4)
    at exports.default (C:\Trader\Logic\node_modules\binance-api-node\dist\http.js:235:15)
    at new exports.default (C:\Trader\Logic\node_modules\binance-api-node\dist\index.js:21:42)
    at C:\Trader\Logic\Trader_Crypto.js:74:53
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:511:3)
```

Downgraded back to 0.5.0, working again.

**Comment by balthazar** [comment]
Need to install `babel-polyfill` for async support.

**Comment by wannesdemaeght** [comment]
Installed babel-polyfill. same error.

**Comment by balthazar** [comment]
[here](http://bfy.tw/GBde).

**Comment by wannesdemaeght** [comment]
oh wow, what a great attitude.
I realise that you are doing this for free, open source and all.
I genuinely thought you would appreciate feedback, to know that your efforts don't go unnoticed.
Guess I was wrong.

anyhow, babel-polyfill IS installed, but that does not make a difference.
Do I need to make a change to my code to reference it explicitly? (I would think no, but not sure at all)

**Comment by balthazar** [comment]
So even after me sending you a link to direct you to information you should have been able to find in 10 second of search, you still require direction, like, really? As you said this is open-source, maintainers expect users to at least try before opening an issue, my attitude is not the problem here but your apparent laziness.

```js
import 'babel-polyfill'
```

**Comment by wannesdemaeght** [comment]
Hey man, nevermind.
I'll just stay on 0.5.0, no worries.

(you do know that import is not natively supported yet, right?)
(you also probably know that if you add code-breaking features, like this obviously is, you might want to add a simple update to your manual?)

**Comment by balthazar** [comment]
A `require` is also possible, but you're probably right though, I'll just use a promise to support more environments without specific configuration.

**Comment by wannesdemaeght** [comment]
anyhow, thank you for your time and effort in making this library available to everyone.
Even noobs like me :-)

**Comment by balthazar** [comment]
Better solution always wins, sorry for the beef, fixed in `0.5.3` 🙂 ❤️

**Comment by wannesdemaeght** [comment]
Working perfectly!
